teuthology/task/install/redhat.py

```diff
@@ -205,14 +205,13 @@ def uninstall_pkgs(ctx, remote, downstream_config):
     :param remote: the teuthology.orchestra.remote.Remote object
     :param downstream_config the dict object that has downstream pkg info
     """
-    rh_rpm_pkgs = downstream_config.get('pkgs').get('rpm')
-    rpm_pkgs = str.join(' ', rh_rpm_pkgs)
-
-    rh_deb_pkgs = downstream_config.get('pkgs').get('deb')
-    deb_pkgs = str.join(' ', rh_deb_pkgs)
 
     if remote.os.name == 'rhel':
+        rh_rpm_pkgs = downstream_config.get('pkgs').get('rpm')
+        rpm_pkgs = str.join(' ', rh_rpm_pkgs)
         remote.run(args=['sudo', 'yum', 'remove', run.Raw(rpm_pkgs), '-y'])
```

Comment:
Instead of this, I would suggest to do the following for simplification:
```
pkgs = downstream_config.get('pkgs').get('rpm')
if pkgs:
    remote.sh(['sudo', 'yum', 'remove'] + pkgs + ['-y'])
```
And similar code for deb.

Reply:
Thanks for the suggestion. I will implement the same for deb also.

Reply:
its not part of this PR but if you fixing it push the new changes, otherwise LGTM

Reply:
Modified the code and  verified the code as suggested.
Log- http://pulpito.ceph.redhat.com/skanta-2021-01-15_19:45:26-rados:singleton-master-distro-basic-clara/